main.tf

```diff
@@ -1,14 +1,14 @@
 resource "azurerm_storage_account" "storage-account" {
-  name                            = "${lower(var.project)}${lower(var.stage)}stateacc"
-  resource_group_name             = var.resource_group
-  location                        = var.location
-  account_kind                    = "StorageV2"
-  account_tier                    = "Standard"
-  account_replication_type        = "ZRS"
-  enable_https_traffic_only       = true
-  allow_nested_items_to_be_public = false
-  min_tls_version                 = "TLS1_2"
-  tags                            = var.tags
+  name                       = "${lower(var.project)}${lower(var.stage)}stateacc"
+  resource_group_name        = var.resource_group
+  location                   = var.location
+  account_kind               = "StorageV2"
+  account_tier               = "Standard"
+  account_replication_type   = "ZRS"
+  enable_https_traffic_only  = true
+  https_traffic_only_enabled = true
```

Comment:
People's front of Judea vs. Judean people's front? ;)
What i mean: Line 8 and 9 are "sinngemäß" the same, do we really need both parameters?

Reply:
Oh my, the actual change should be that one of that was removed because it wasn't supported in ARM v4 🤦🏽

Sorry, will check and remove the conflicting one.